content/2016-10-16-haskell-servant-template.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+tags: haskell prometheus servant docker
+date: 2016-10-16
+title: servant-template: production-ready Haskell web services in 5 minutes
+author: jml
+---
+
+If you want to write a web API in Haskell, then you should start by using my
+new cookiecutter template at https://github.com/jml/servant-template. It'll
+get you a production-ready web service in 5 minutes or less.
+
+For some values of "production". And "ready".
+
+Whenever you start any new web service and you actually care about getting it
+working and available to users, you need to get a few things sorted out:
```

Comment:
I'm quite allergic to "you need to", I usually try to phrase it a bit more positively, e.g. "it's useful to have"